src/data-types.js

```diff
@@ -573,7 +573,7 @@ class DATEONLY extends ABSTRACT {
 }
 
 /**
- * A key / value store column. Only available in Postgres.
+ * A key / value store column. Only available in SQLite, MySQL, MariaDB and PostgreSQL.
```

Comment:
From my recent research, hstore is only available in Postgres. Do you have a link to the documentation of sqlite/mysql/mariadb for hstore? https://new.sequelizejs.com/docs/v7/other-topics/other-data-types#miscellaneous-datatypes